RHEL/7/input/xccdf/system/accounts/session.xml

```diff
@@ -46,12 +46,13 @@ left unattended.
 </description>
 <rationale>
 </rationale>
-<ocil clause="value of TMOUT is not greater than or equal to expected setting">
+<ocil clause="value of TMOUT is not less than or equal to expected setting">
```

Comment:
Are you sure about this change? The timeout has to be lower, or "not greater". I think the original clause was correct.